cpp/examples/sbml2memilio.h

```diff
@@ -0,0 +1,856 @@
+#include <cmath>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+
+#include <sbml/SBMLTypes.h>
+
+LIBSBML_CPP_NAMESPACE_USE
+
+/*
+    @brief Get the filename
+
+    @param filename The filename to be processed
+
+    Uses `boost::filesystem::path` to get the path to the input file, then uses `stem()` to get the name of the input file without file ending.
+*/
+std::string get_filename(const char* filename)
```

Comment:
the doc for each function is nice. However, there could be more comment about the implication into the overall context.